cmd/lncli/commands.go

```diff
@@ -1093,15 +1323,16 @@ func getNodeInfo(ctx *cli.Context) error {
 
 var QueryRouteCommand = cli.Command{
 	Name:        "queryroute",
-	Usage:       "queryroute --dest=[dest_pub_key] --amt=[amt_to_send_in_satoshis]",
-	Description: "queries the channel router for a potential path to the destination that has sufficient flow for the amount including fees",
+	Usage:       "Query a route to a destination.",
+	Description: "Queries the channel router for a potential path to the destination that has sufficient flow for the amount including fees",
+	ArgsUsage:   "amt dest",
```

Comment:
Minor nit, but I'd prefer for the `dest` to be accepted as the first argument rather than the second.